test/controllers/beneficial.owner.delete.warning.controller.spec.ts

```diff
@@ -198,4 +244,124 @@ describe("BENEFICIAL OWNER DELETE WARNING controller", () => {
       expect(resp.text).toContain(SERVICE_UNAVAILABLE);
     });
   });
+
+  describe("POST with url params tests", () => {
+    test("redirects to the beneficial owner type page if No option has been selected", async () => {
+      const resp = await request(app)
```

Comment:
Wondering if you should be making the same assertions as you made on lines 274 and 277 below, which would probably mean needing to do the mock call you did on line 263 too.

Reply:
No, the flag condition is within the scope of the condition for 0 or 1.